helper.py

```diff
@@ -0,0 +1,244 @@
+import logging
+import os
+import glob
+import sys
+from datetime import datetime
+import pandas as pd
+
+from dotenv import load_dotenv
+
+
+def processSrtFile(srtFile):
+    """
+    Process an SRT file and extract the transcript data.
+
+    Args:
+        srtFile (str or list): The path to the SRT file or a list of paths.
+
+    Returns:
+        pandas.DataFrame: A DataFrame containing the extracted transcript data with columns 'Line', 'Start', and 'End'.
+    """
+    if type(srtFile) == list:
+        srtFile = srtFile[0]
+
+    with open(srtFile, "r") as f:
```

Comment:
This could cause a file not found error. so need some action to handle the situation. So the code should be know how to handle the happy and unhappy path

Reply:
Noted. This will be handled in a future PR, while handling #11.